_posts/2020-08-26-hhvm-4.72.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: "HHVM 4.72"
+layout: post
+author: fred
+category: blog
+-
```

Comment:
This renders differently from before, is the formatting correct here? (maybe needs to be `---` instead of `-`?)

Reply:
ah, that's why there was a random `--` lower down the page :D